coreos-koji-tagger/coreos_koji_tagger.py

```diff
@@ -271,13 +271,17 @@ def process(self, message: fedora_messaging.api.Message):
             logger.info(f'Skipping message from unrelated branch: {branch}')
             return
 
-        # Now grab data from the commit we should operate on:
-        # XXX: should update for multi-arch
+        # Now grab lockfile data from the commit we should operate on:
         commit = msg['head_commit']['id']
-        url = f'https://raw.githubusercontent.com/{repo}/{commit}/manifest-lock.x86_64.json'
-        logger.info(f'Attempting to retrieve data from {url}')
-        r = requests.get(url)
-
+        desiredrpms = set()
+        for arch in ['x86_64', 'aarch64', 'ppc64le', 's390x']:
+            for lockfile in ['manifest-lock']:
+                url = f'https://raw.githubusercontent.com/{repo}/{commit}/{lockfile}.{arch}.json'
+                logger.info(f'Attempting to retrieve data from {url}')
+                r = requests.get(url)
+                # parse the lockfile and add the set of rpm NEVRAs (strings)
+                if r.ok:
```

Comment:
I think we should differentiate between `404` and any other error.  At least log non-404.

Reply:
Yeah Let's log errors. Really even 404 shouldn't happen once we have all arches in place, so let's log that one too.

Reply:
should be fixed now